src/Symfony/Component/PropertyAccess/Annotation/Property.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\PropertyAccess\Annotation;
+
+/**
+ * Property accessor configuration annotation.
+ *
+ * @Annotation
+ * @Target({"PROPERTY"})
+ *
+ * @author Luis Ramón López <lrlopez@gmail.com>
+ */
+class Property
```

Comment:
I agree with @webmozart, it's weird to have different way to do the same thing. What about having only a `@Property` annotation with an optional `name` attribute in case you use it on a method?